data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -664,7 +665,7 @@
                                                                     border_size = 5
                                                                     [label]
                                                                         definition = "gold_small"
-                                                                        label = "WC2 version"
+                                                                        label = _ "WC2 version"
```

Comment:
Since it's going translatable, let's also remove the 2 while we're at it.

Maybe we can also spell it out in full – "World Conquest".

There may also be an argument that an independent version isn't needed, but that's probably beyond the scope of this PR.